Source/gs/GSHandler.cpp

```diff
@@ -611,6 +611,9 @@ void CGSHandler::WriteRegisterImpl(uint8 nRegister, uint64 nData)
 
 	switch(nRegister)
 	{
+	case GS_REG_ST:
+		m_nReg[nRegister] &= 0xFFFFFF00FFFFFF00;
```

Comment:
These (ST) are floating point values. Does it makes sense to mask out the last 8 bits of these values?

Reply:
Yep, thats what the GS manual explicitly states.